sqlfs/table.go

```diff
@@ -8,20 +8,27 @@ import (
 // createTable creates a table, if it doesn't exist.  If the table
 // name includes the database name, e.g., "db.tbl", it creates the
 // database if necessary.
-func createTable(db *sql.DB, table string) error {
-	// FIXME(tony): HIVE and ODPS don't support AUTO_INCREMENT
-	// FIXME(tony): Hive and ODPS don't support BLOB, use BINARY instead
-	stmt := fmt.Sprintf("CREATE TABLE IF NOT EXISTS %s (id INT AUTO_INCREMENT, block BLOB, PRIMARY KEY (id))", table)
+func createTable(db *sql.DB, driver, table string) error {
+	// HIVE and ODPS don't support AUTO_INCREMENT
+	// Hive and ODPS don't support BLOB, use BINARY instead
+	var stmt string
+	if driver == "mysql" || driver == "sqlite3" {
```

Comment:
Maybe we need `DriverType` type to unify these checks in order to avoid things like coding mistakes like `"Hive" == "hive"`

Reply:
Good catch.
This will be fixed in [Unify DriverType name](https://github.com/sql-machine-learning/sqlflow/issues/374).